### DISCUSSION TITLE
distill-style in-line math color in dark mode

### DISCUSSION THREAD

**Discussion by glenntfung** [alshedivat/al-folio#2909] [General]
As shown in [this distill-style post example section](https://alshedivat.github.io/al-folio/blog/2021/distill/#equations), if dark mode is enabled, the color of the in-line equations would not turn white. However, the display math is fine. 

<img width="736" alt="image" src="https://github.com/user-attachments/assets/a1e9f06c-fda4-4533-ad36-437edc59a64a" />

I think this should be related to [_sass/_distill.scss](https://github.com/alshedivat/al-folio/blob/main/_sass/_distill.scss), but I am not sure what should be changed.

**Comment by george-gca** [comment]
Actually this happens because the distill template for this case with `$...$` imports another css, which overwrites the color part. You can see this happening with the browser debug.

Weirdly enough, adding

```css
span.katex {
  color: var(--global-text-color) !important;
}
```

to `_sass/_distill.scss` doesn't solve this. Can you open a bug for this? Maybe someday someone can add a fix for this.

**Reply by glenntfung** [reply]
Sure, already done!
